### PR TITLE
Update documentation for the new Entity header

### PR DESCRIPTION
**Pull request by bhashkarsharma**
## Give us some context

* What problems does this PR solve?
  * This PR updates the documentation to match the new Entity header. It is related to [another PR](https://github.com/newrelic/developer-website/pull/2238) on the `developer-website`. Please refer to the **Reviewer Notes** on that PR for more details.
* Add any context that will help us review your changes such as testing notes, links to related docs, screenshots, etc.
  * Screenshots of new header with metadata and tags widgets:
Screenshot 2023-07-20 at 17 37 29
Screenshot 2023-07-20 at 17 58 34

* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx [hunk 1]
```diff
@@ -125,14 +125,14 @@ Click a dashboard to open its charts. A dashboard page includes these features:
     id="dash-json"
     title="Import your dashboard as JSON"
   >
-    You can import your dashboard as JSON code and add it to the clipboard. Click the <Icon name="fe-more-horizontal"/> icon on the right corner and select **Copy JSON to clipboard**. See how to [import a dahsboard](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-import).
+    You can import your dashboard as JSON code and add it to the clipboard. Click the <Icon name="fe-more-horizontal"/> icon on the right corner and select **Copy JSON to clipboard**. See how to [import a dashboard](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-import).
```
Comment (bhashkarsharma) [inline src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:128]: Fixed a typo.

**Comment by bradleycamacho** [comment]
Hi @bhashkarsharma , thanks for opening this PR! Could you sign the CLA by clicking the yellow button above so I can merge this in once I review? Thanks!

**Comment by bhashkarsharma** [comment]
Hi @bradleycamacho, I have signed the CLA. Thanks!

**Review by nbaenam — APPROVED** [review]
Thank you for the update! I've added some TW edits and updated all the screenshots

**Pull request merged**